pkg/component/etcd/bootstrap.go

```diff
@@ -255,6 +258,29 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 			},
 		}
 
+		service = &corev1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      druidServiceName,
+				Namespace: b.namespace,
+				Labels: utils.MergeStringMaps(map[string]string{
+					resourcesv1alpha1.HighAvailabilityConfigType: resourcesv1alpha1.HighAvailabilityConfigTypeController,
+				}, labels()),
+			},
+			Spec: corev1.ServiceSpec{
+				Type:            corev1.ServiceTypeClusterIP,
+				SessionAffinity: corev1.ServiceAffinityNone,
+				Selector:        labels(),
+				Ports: []corev1.ServicePort{
+					{
+						Name:       "metrics",
+						Protocol:   corev1.ProtocolTCP,
+						Port:       metricsPort,
+						TargetPort: intstr.FromInt(int(metricsPort)),
+					},
+				},
+			},
+		}
```

Comment:
I guess you need

```golang
utilruntime.Must(gardenerutils.InjectNetworkPolicyAnnotationsForSeedScrapeTargets(service, portMetrics))
```

Also, unit tests must be adapted.

Reply:
Okay. I will add it. Yes, the unit test is failing. I made necessary changes in the unit test file locally but strange error is occurring. Trying to fix that now.

Reply:
Hi Rafael, I made the changes you suggested and fixed the unit test case. All unit test cases are passing in my local machine